src/App.css

```diff
@@ -55,3 +55,7 @@ td {
 td:hover {
   background-color: tomato;
 }
+
+.winner {
+  background-color: turquoise;
```

Comment:
...IMO, I prefer hex codes than this

Reply:
Agreed. Most of this .css file is going to be deleted on the next PR and replaced with Styled Components :D